_layouts/organizers.html

```diff
@@ -33,7 +33,9 @@
     <body>
         <div class="reveal">
             <div class="slides">
-                {{ content }}
+                <section>
```

Comment:
This change makes the slides vertical, so we can use a clicker to navigate them

Reply:
Oh right